src/channels/web/static/index.html

```
@@ -304,12 +304,16 @@ <h3>Installed Skills</h3>
           </div>
         </div>
         <div class="extensions-section">
-          <h3>Install Skill by URL</h3>
+          <h3>Install Skill</h3>
           <div class="ext-install-form">
             <input type="text" id="skill-install-name" placeholder="Skill name or slug">
-            <input type="text" id="skill-install-url" placeholder="HTTPS URL to SKILL.md (optional)">
+            <input type="text" id="skill-install-url" placeholder="HTTPS URL to SKILL.md or .skill">
             <button onclick="installSkillFromForm()">Install</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)